python/pyspark/sql/utils.py

```diff
@@ -116,22 +116,6 @@ def require_test_compiled() -> None:
         )
 
 
-def require_minimum_plotly_version() -> None:
```

Comment:
Hm, don't we also support plot in the plain DataFrame.plot too? Then it should stay here

Reply:
cc @xinrong-meng

Reply:
this `require_minimum_plotly_version` is only used for plain DF.plot?

Reply:
make sense, let me keep it in the file